Dockerfile

```diff
@@ -1,98 +1,190 @@
-# syntax=docker/dockerfile:1.4
-# This needs to be bullseye-slim because the Ruby image is built on bullseye-slim
-ARG NODE_VERSION="16.18.1-bullseye-slim"
-
-FROM ghcr.io/moritzheiber/ruby-jemalloc:3.0.4-slim as ruby
-FROM node:${NODE_VERSION} as build
-
-COPY --link --from=ruby /opt/ruby /opt/ruby
-
-ENV DEBIAN_FRONTEND="noninteractive" \
-    PATH="${PATH}:/opt/ruby/bin"
-
+# Ignoreing these here since we don't want to pin any versions and the Debian image removes apt-get content after use
+# hadolint global ignore=DL3008
+FROM ruby:3.0.5-slim-bullseye
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
-WORKDIR /opt/mastodon
-COPY Gemfile* package.json yarn.lock /opt/mastodon/
-
-# hadolint ignore=DL3008
-RUN apt-get update && \
-    apt-get install -y --no-install-recommends build-essential \
-        ca-certificates \
-        git \
-        libicu-dev \
-        libidn11-dev \
-        libpq-dev \
-        libjemalloc-dev \
-        zlib1g-dev \
-        libgdbm-dev \
-        libgmp-dev \
-        libssl-dev \
-        libyaml-0-2 \
-        ca-certificates \
-        libreadline8 \
-        python3 \
-        shared-mime-info && \
-    bundle config set --local deployment 'true' && \
-    bundle config set --local without 'development test' && \
-    bundle config set silence_root_warning true && \
-    bundle install -j"$(nproc)" && \
-    yarn install --pure-lockfile --network-timeout 600000
-
-FROM node:${NODE_VERSION}
+# https://github.com/nodejs/docker-node/blob/main/16/bullseye-slim/Dockerfile
```

Comment:
Not sure I follow the benifit of copy/pasting the official Node image text and taking on the burden of keeping it in sync, even if we don't change the template often at all

Reply:
@nschonni Thanks to review 😀

I tried to use official image instructions for keeping image light and using specific version of nodejs, but yeah that can be problem.
But, As I know, official didn't change that template either, they just changes version args to release new version.

But if this is not a prefer way, than I might use nvm or other node version manager to install specific version easily.

Reply:
@nschonni I change to use [NodeSource's binaries](https://github.com/nodesource/distributions).